src/scene/graph-node.js

```diff
@@ -43,6 +43,8 @@ Object.assign(pc, function () {
         this.normalMatrix = new pc.Mat3();
         this._dirtyNormal = true;
 
+        this._worldTransformDrivenByPhysics = false;
```

Comment:
I might be missing something, this seems to be not used here and in any other pull.